.prettierrc.js

```diff
@@ -0,0 +1,6 @@
+"use strict";
```

Comment:
the `.prettierrc.js` format doesn't work with the vscode extension :( 

mind using `.prettierrc` with the yaml format which does?

Reply:
Sure thing, good to know

Reply:
Thank you